chatbot.py

```diff
@@ -93,6 +104,8 @@
                 print("Could not understand audio")
                 engine.say('I didnt get that. Rerun the code')
                 engine.runAndWait()
+    intent = interpreter.parse(translate)
+    print(intent['intent'])
```

Comment:
no need to print intent.

Reply:
I just added it to check the intent. I will remove it.